example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -1,13 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
 
-    <application
-        android:usesCleartextTraffic="true"
-        tools:targetApi="28"
-        tools:ignore="GoogleAppIndexingWarning">
-        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
```

Comment:
@IjzerenHein Is it necessary to remove the DevSettingsActivity? I guess the upgrade of Flipper's version resolves the issue?

Reply:
I guess it's not @tvanlaerhoven for the flipper issue. I happened to notice it when comparing this project to a vanilla react-native bootstrapped project. That setting was moved to the main `AndroidManifest.xml`, it appeared that the example project had been upgraded at some point and some things had been missed.
You can remove it from this PR, shouldn't matter for the flipper issue.

Reply:
@IjzerenHein Ah yes, it's also in the main manifest file indeed. I'll leave it as such.